types/options.go

```diff
@@ -398,11 +415,12 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 	if len(storeOptions.GraphDriverOptions) == 1 && storeOptions.GraphDriverOptions[0] == "" {
 		storeOptions.GraphDriverOptions = nil
 	}
+	return nil
 }
 
-func Options() StoreOptions {
-	defaultStoreOptionsOnce.Do(loaddefaultStoreOptions)
-	return defaultStoreOptions
+func Options() (StoreOptions, error) {
+	defaultStoreOptionsOnce.Do(loadDefaultStoreOptions)
+	return defaultStoreOptions, loaddefaultStoreOptionsErr
```

Comment:
Is this a typo?

Reply:
I messed it up during a rebase, fixed now